src/extension/ncwms/src/test/java/au/org/emii/geoserver/wms/URLIndexWFSHttpTest.java

```diff
@@ -0,0 +1,163 @@
+package au.org.emii.geoserver.wms;
+
+import junit.framework.TestCase;
+import org.apache.commons.io.IOUtils;
+import org.easymock.EasyMock;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.ByteArrayInputStream;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ URL.class })
+public class URLIndexWFSHttpTest extends TestCase {
+    private URLIndexWFSHttp urlIndexWFSHttp = null;
```

Comment:
This variable appears to be unused. If it is then there setUp() method can also go (and the import statement for the Before annotation)